src/Controller/MenuController.java

```diff
@@ -195,6 +196,13 @@ public void main() {
 	 * Display the Study Dashboard pane.
 	 */
 	public void loadDashboard() {
+		
```

Comment:
This trailing white space needs to be removed.

Reply:
This issue has been fixed and pushed.